docs/source/reference/cmd.rst

```diff
@@ -0,0 +1,48 @@
+.. _cmd:
+
+ChainerUI command manual
+========================
+
+.. _cmd_server:
+
+Server
+------
+
+Run ChainerUI server. To stop, press ``Ctrl+C`` on the console::
+
+  $ chainerui server
+
+* ``--host`` or ``-H``: (optional) set original host name
+* ``--port`` or ``-p``: (optional) set original port number, set ``5000`` on default
+* ``--debug`` or ``-d``: (optional) run server with debug mode
+
+
+.. _cmd_database:
+
+Database
+--------
+
+Create ChainerUI database. ChainerUI creates ``~/.chainerui/db/chainerui.db`` on default and database references the file::
+
+  $ chainerui db create
+
+Setup schema for ChainerUI. ``upgrade`` operation is always necessary when create new database or change the schema on version up::
```

Comment:
Setup the schema for ChainerUI. The upgrade operation is always necessary when creating a new database or changing...